### PR TITLE
Fixes double-header on 404 pages

### PR DESCRIPTION
**Pull request by andreban**
- Makes 404.hbs compatible with the contentOnly templates
- Change app.js to pass the contentOnly parameter to 404.hbs

### CODE REVIEW

**Comment by andreban** [comment]
Fixes #416

File: views/404.hbs [hunk 1]
```diff
@@ -11,18 +11,24 @@
  See the License for the specific language governing permissions and
  limitations under the License. -->
 
+{{#unless contentOnly}}
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>Gulliver - PWA Directory - File Not Found</title>
```
Comment (samthor) [inline views/404.hbs]: I presume removing the title is intentional.

File: app.js [hunk 2]
```diff
@@ -118,7 +118,8 @@ app.use(require('./controllers'));
 
 // If no route has matched, return 404
 app.use((req, res) => {
-  res.status(404).render('404.hbs', {nonce1: req.nonce1, nonce2: req.nonce2});
+  res.status(404).render('404.hbs',
```
Comment (samthor) [inline app.js]: trailing whitespace is breaking Travis CI

**Comment by samthor** [comment]
LGTM

**Comment by juliantoledo** [comment]
LGTM, I fixed the trailing space in the merge.

**Pull request merged**